### PR TITLE
hottext: init at 1.3

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change

Utility for speed-reading.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 122178` at 331fc29a run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/63c71c0650857f4a492a9df496898437">hottext</a></li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`buildPhase` should probably contain `runHook preBuild` and `runHook postBuild`.

Near pkgs/tools/text/hottext/default.nix:66:3:
```
   |
66 |   buildPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/tools/text/hottext/default.nix:71:3:
```
   |
71 |   installPhase = "install -Dt $out/bin src/$pname";
   |   ^
```
</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

File: pkgs/tools/text/hottext/default.nix [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  installPhase = "install -Dt $out/bin src/$pname";
+
+  meta = with lib; {
+    description = "Simple RSVP speed-reading utility";
+    longDescription = ''
+    '';
+    license = licenses.unlicense;
+    homepage = "https://git.sr.ht/~ehmry/hottext";
+    maintainers = with maintainers; [ ehmry ];
+    inherit (SDL2.meta) platforms;
```
Comment (dotlambda) [inline pkgs/tools/text/hottext/default.nix:80]:
```suggestion
```

**Review by dotlambda — DISMISSED** [review]

File: pkgs/tools/text/hottext/default.nix (hunk 1 above)
Comment (ehmry) [reply pkgs/tools/text/hottext/default.nix:80]: Why?

Comment (dotlambda) [reply pkgs/tools/text/hottext/default.nix:80]: There's no point: `SDL2` is in the inputs so `hottext` can only be built where `SDL2` is anyway.

Comment (ehmry) [reply pkgs/tools/text/hottext/default.nix]: Got it.

**Comment by ehmry** [comment]
Fixed build for x86, added "desktop item" file.

**Pull request merged**